docs/using/tutorial/movingvolume.rst

```diff
@@ -0,0 +1,231 @@
+.. _tutorial-moving-a-data-volume:
+
+==============================
+Tutorial: Moving a Data Volume
+==============================
+
+In this tutorial (30 minutes)
+-----------------------------
+
+You will use Flocker to migrate a Docker container with its data volume from one host to another.
+The container you move will be part of a two-container application, the other container will not move and the two will remain connected even when they are on different hosts.
+
+To begin the tutorial you will first install the Flocker client on your local machine, then install Flocker onto two hosts.
+You will then be ready to use Flocker to migrate a Docker container with a volume attached from one host to the other.
+
+.. note:: This tutorial takes roughly 30 minutes, but because there are a few things to download, times might vary depending on the speed of your connection.
+
+You will need
+-------------
+
+1. Somewhere to install the Flocker client.
+Make sure you have **one** of the following on your machine:
+
+- OS X with `Homebrew <http://brew.sh/>`_ installed.
+- Ubuntu 14.04.
+
+2. Two hosts for two instances of Flocker.
+The options are:
+
+- Two Virtual Machines (VMs) on your local machine.
+  For this tutorial, you are supplied with Vagrant images to create the tutorial environment on VMs using VirtualBox, so you must have `Vagrant <https://www.vagrantup.com/>`_ and `VirtualBox <https://www.virtualbox.org/>`_ installed.
+- AWS or Rackspace (you will need an account with root access).
+- Physical hosts with a supported operating system.
+- Any combination of the above.
```

Comment:
The combination option isn't tested and is not easily followed in this document e.g. if you use the Vagrantfile you end up with two nodes, which is the control node? What do you do differently with the control node? I'd remove this option.